sentry_streams/tests/test_runner.py

```diff
@@ -55,5 +72,8 @@ def test_iterate_edges(create_pipeline: Pipeline) -> None:
             call.filter(create_pipeline.steps["step2"], ANY),
             call.map(create_pipeline.steps["step3"], ANY),
             call.map(create_pipeline.steps["step4"], ANY),
+            call.router(create_pipeline.steps["step5"], ANY),
+            call.branch(create_pipeline.steps["step6"], ANY),
+            call.branch(create_pipeline.steps["step7"], ANY),
```

Comment:
These should not be called. If they are we have a problem in the way iterate_edges works.